### PR TITLE
RELATED: RAIL-1277 properly detect horizontal scrollbar

### PR DESCRIPTION
**Pull request by no23reason**
Checked in
* https://github.com/gooddata/gdc-app-components/pull/485 ✅ 
* https://github.com/gooddata/gdc-analytical-designer/pull/2022 ✅
* https://github.com/gooddata/gdc-dashboards/pull/1614 ✅

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3073/ : SUCCESS in 9s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2426/ : FAILURE in 6m 28s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3074/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2427/ : SUCCESS in 2m 20s

File: src/components/core/PivotTable.tsx [hunk 1]
```diff
@@ -910,9 +915,16 @@ export class PivotTableInner extends BaseVisualization<IPivotTableInnerProps, IP
         }
         const rowHeight = this.getRowHeight();
         const headerHeight = (this.gridApi as any).headerRootComp.eHeaderContainer.clientHeight;
-        const bodyHeight = rowCount * rowHeight;
+        const bodyHeight = rowCount * rowHeight + 1;
```
Comment (tomasreichmann) [inline src/components/core/PivotTable.tsx]: This could use a comment

**Comment by tomasreichmann** [comment]
LGTM

**Comment by gdgate** [comment]
Build in pipeline gate aborted.

File: src/components/core/PivotTable.tsx (hunk 1 above)
Comment (no23reason) [reply src/components/core/PivotTable.tsx]: True, I added it. Thanks!

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/660/ : SUCCESS in 24s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2430/ : SUCCESS in 3m 22s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1275/ : UNSTABLE in 18m 23s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/642/ : SUCCESS in 30m 36s

**Comment by no23reason** [comment]
extended test - storybook

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1276/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3076/ : SUCCESS in 15s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/760/ : UNSTABLE in 22m 19s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/661/ : SUCCESS in 1m 18s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2431/ : SUCCESS in 17m 05s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1277/ : SUCCESS in 19m 27s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/643/ : SUCCESS in 25m 22s

**Pull request merged**